src/main/java/com/glencoesoftware/omero/zarr/ZarrPixelBuffer.java

```diff
@@ -137,8 +137,10 @@ public ZarrPixelBuffer(Pixels pixels, Path root, Integer maxPlaneWidth,
                     int w = key.get(6);
                     int h = key.get(7);
                     int[] shape = new int[] { 1, 1, 1, h, w };
+                    // Check planar read size (sizeX and sizeY only)
+                    checkReadSize(Arrays.copyOfRange(shape, 3, 5));
```

Comment:
I assume this plane size check could also have been introduced at the beginning of `getTile` alongside the `checkBounds`. Is there any particular advantage either way?

Reply:
I don't think there's a particular advantage either way and I was torn on which one was "better". Having some time to think about it, my current thought is it's better to do it in getTile rather than here. It's cleaner.